assets/css/cursor.css

```diff
@@ -0,0 +1,32 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400|Raleway:300);
```

Comment:
Why do we need a font for a stylesheet regarding a cursor?

Reply:
Thanks for pointing it out! , I will remove it.